File: Frontend/ClientApp/app/hoshin-quality/tasks/models/Task.ts
import {EditTaskEvidence} from './EditTaskEvidence';
import { User } from "./User";

export class Task{
       taskID : number;
       entityID: number;
       entityType: number;
       description: string;
       responsibleUserID: string;
       implementationPlannedDate: Date;
       implementationEffectiveDate:Date;
       observation: string;
       result: string;
       taskStateID: number;
       requireEvidence: boolean;
       taskEvidences: EditTaskEvidence[];
       filetodelete:string[];
       responsibleUser: User;
       typeButtonSubmit: string;
       codeState: string;
       taskState;
}

File: src/Hoshin.Quality.Domain/Task/Task.cs

﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;
using Hoshin.Core.Domain;
using Hoshin.Core.Domain.Users;
using Hoshin.Quality.Domain.Interfaces;



namespace Hoshin.Quality.Domain.Task
{
    public class Task
    {
      
        public int TaskID { get; set; }
        public int EntityID { get; set; }
        public int EntityType { get; set; }
        public string Description { get; set; }
        public string ResponsibleUserID { get; set; }
        public User ResponsibleUser { get; set; }
        public DateTime ImplementationPlannedDate { get; set; }
        public DateTime? ImplementationEffectiveDate { get; set; }
        public string Observation { get; set; }
        public string Result { get; set; }
        public int TaskStateID { get; set; }
        public bool RequireEvidence { get; set; }
        public TaskState.TaskState TaskState { get; set; }
        public List<Evidence.Evidence> TaskEvidences { get; set; }
        public List<string> DeleteEvidencesUrls { get; set; }
        public List<string> NewEvidencesUrls { get; set; }
        public string FirstName { get; set; }
        public string Surname { get; set; }
        public int PlantID { get; set; }
        public int SectorID { get; set; }
        public DateTime? overdureTime { get; set; }
        public Task()
        {

        }
    }




}


